test_scripts/API/SeparatingChangeAudibleStatus/003_PHONE_CALL_events_during_audio_video.lua

```diff
@@ -0,0 +1,120 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md
+---------------------------------------------------------------------------------------------------
+-- Description:
+-- Check the processing of separating the change of Audible status and the change of HMI Status on 'PHONE_CALL' events
+-- during audio/video streaming
+-- In case:
+-- 1) There is a mobile app which is audio/video source
+-- 2) App starts Audio/Video streaming
+-- 3) Mobile App is moving to another state by 'PHONE_CALL' event (isActive = true) from HMI
+-- SDL does:
+--  - send OnHMIStatus notification with 'audioStreamingState'= "NOT_AUDIBLE" parameter
+--  - send Navigation.OnAudioDataStreaming(available=false)
+--  - not send Navi.OnAudioDataStreaming(false) notification to HMI
+-- In case:
+-- 4) Mobile App is moving to another state by 'PHONE_CALL' event (isActive = false) from HMI
+-- SDL does:
+--  - send OnHMIStatus notification with 'audioStreamingState'= "AUDIBLE" parameter
+--  - send Navigation.OnAudioDataStreaming(available=true)
+--  - not send Navi.OnAudioDataStreaming(false) notification to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/SeparatingChangeAudibleStatus/common')
+
+--[[ Local Variables ]]
+local appId = 1
+
+local testCases = {
+  [001] = { t = "PROJECTION",  m = true,  s = {
+    [1] = { e = common.userActions.phoneCallStart,   l = "FULL",       a = "NOT_AUDIBLE",  v = "STREAMABLE" },
+    [2] = { e = common.userActions.phoneCallEnd,     l = "FULL",       a = "AUDIBLE",      v = "STREAMABLE" },
+    [3] = { e = common.userActions.deactivateApp,    l = "LIMITED" }, -- a = "AUDIBLE",      v = "STREAMABLE"
```

Comment:
I would propose to uncoment this code otherwise within fix suggested for common.checkHMIStatus() script would fail